extensions/BMO/lib/Reports/Groups.pm

```diff
@@ -185,18 +185,22 @@ sub members_report {
     ? map { lc($_->name) } Bugzilla::Group->get_all
     : _get_permitted_membership_groups();
 
-  my $include_disabled = $cgi->param('include_disabled') ? 1 : 0;
-  $vars->{'include_disabled'} = $include_disabled;
-
   # don't allow all groups, to avoid putting pain on the servers
   my @group_names
     = sort grep { !/^(?:bz_.+|canconfirm|editbugs|editbugs-team|everyone)$/ }
     @grouplist;
   unshift(@group_names, '');
   $vars->{'groups'} = \@group_names;
 
+  my $include_disabled = $cgi->param('include_disabled') ? 1 : 0;
+  $vars->{'include_disabled'} = $include_disabled;
+
+  my $search_type = $cgi->param('search_type') || 'all';
```

Comment:
Can we limit `search_type` to only a set of allowed values here? Or perhaps in the `members_complete` subroutine.

Reply:
Sure. Will do that.